certified-connectors/DocumentTranslator/apiDefinition.swagger.json

```diff
@@ -0,0 +1,907 @@
+          },
+          "default": {
+            "description": "Operation Failed."
+          }
+        },
+        "summary": "Get supported glossary formats",
+        "description": "Gets a list of supported glossary formats.",
+        "operationId": "GetSupportedGlossaryFormats",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "The value is the Azure key for your Translator or Cognitive Services resource.",
+            "x-ms-summary": "The value is the Azure key for your Translator or Cognitive Services resource.",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/translator/text/batch/v1.0/storagesources": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "value"
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Operation Failed."
+          }
+        },
+        "summary": "Get supported storage sources",
+        "description": "Gets a list of supported storage sources/options.",
+        "operationId": "GetSupportedStorageSources",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "The value is the Azure key for your Translator or Cognitive Services resource.",
+            "x-ms-summary": "The value is the Azure key for your Translator or Cognitive Services resource.",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/translator/text/batch/v1.1/batches/{id}/documents": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "path": {
+                        "type": "string",
+                        "description": "path"
+                      },
+                      "sourcePath": {
+                        "type": "string",
+                        "description": "sourcePath"
+                      },
+                      "createdDateTimeUtc": {
+                        "type": "string",
+                        "description": "createdDateTimeUtc"

```

Comment:
@HamshavathiM strings in CaMel/caMel case are considered [non-translatable](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings). If you want the value to be localized in the future, the casing should be fixed so that the value looks like a sentence or a single word.